WordPress/gutenberg
# Storybook: Addon to wrap stories in max-width div

Related:
- https://github.com/WordPress/gutenberg/pull/41860#discussion_r999206297

## What?

Adds a quick means to wrap a story in a div reflecting the max-width and padding constraints of the WordPress sidebar.

## Why?

This addon will allow us to remove the ad hoc wrappers on some components, e.g. `BorderBoxControl`, that set a `max-width` matching the WordPress editor sidebars. This way, the component can stretch the full width of the viewport by default illustrating how the component adapts to fill space.

When developing components, this addon will make it quick and easy to constrain components to a real-world scenario, such as the width of our editor sidebars.

## How?

- Creates a simple Storybook toolbar addon with a decorator.
- Doesn't update individual storybook examples for components. That will be done separately.

## Testing Instructions

1. Load Storybook
2. Navigate to the `UnitControl` story and confirm the control stretches the full width of your canvas
3. Toggle on the `WP Sidebar` option for the new "Wrapper" addon.
    <img width="535" alt="Screen Shot 2022-10-20 at 10 55 49 am" src="https://user-images.githubusercontent.com/60436221/196831699-917fa6d3-3b4b-407c-afff-727177e8b72d.png">
4. The UnitControl should now be wrapped in a div with a max-width and padding.


## Screenshots or screencast 


https://user-images.githubusercontent.com/60436221/196831768-8d82a1d6-b718-4762-9356-fb1a9294203b.mp4
